tic-tac-toe/src/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import * as serviceWorker from './serviceWorker';
+
+function Square({ value, onClick }) {
+
+  return (
+    <button className="square" onClick={onClick}>
+      {value}
+    </button>
+  );
+}
+
+ReactDOM.render(<Square />, document.getElementById("root"));
+
+function Restart({ onClick }) {
+
+  return (
+    <button className="restart" onClick={onClick}>
+      Play again
+    </button>
+  );
+}
```

Comment:
The extension of this project with the `<Restart/>` component is groovy and allows for a an intuitive user experience. 👍